dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql

```diff
@@ -2008,10 +2008,14 @@ DROP TABLE IF EXISTS `t_ds_audit_log`;
 CREATE TABLE `t_ds_audit_log` (
   `id` bigint(11) NOT NULL AUTO_INCREMENT COMMENT'key',
   `user_id` int(11) NOT NULL COMMENT 'user id',
-  `resource_type` int(11) NOT NULL COMMENT 'resource type',
-  `operation` int(11) NOT NULL COMMENT 'operation',
-  `time` datetime DEFAULT CURRENT_TIMESTAMP COMMENT 'create time',
-  `resource_id` int(11) NULL DEFAULT NULL COMMENT 'resource id',
+  `object_id` bigint(20) DEFAULT NULL COMMENT 'object id',
+  `object_name` varchar(100) DEFAULT NULL COMMENT 'object id',
+  `object_type` varchar(100) NOT NULL COMMENT 'object type',
```

Comment:
```suggestion
  `object_id` bigint(20) DEFAULT NULL COMMENT 'object id',
  `object_name` varchar(100) DEFAULT NULL COMMENT 'object id',
  `object_type` varchar(100) NOT NULL COMMENT 'object type',
```

Use `domain_id` or `module_id` is better than object_id?

Reply:
For this part, I defined object id means the object I modified, `task instance` it's not a module. `File `is not a domain, I think `object` is more suit for this, can cover all the stuff we record, WDYT